utils/ort/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -239,6 +187,17 @@ private fun replaceYears(copyrightStatement: String): Pair<String, Set<Int>> {
  * TODO: Maybe treat URLs similar to years, e.g. entries which differ only in URLs and years can be merged.
  */
 object CopyrightStatementsProcessor {
+    data class Parts(
```

Comment:
PR: Is there any bigger umbrella this PR (series) is made for. Would you mind sharing the ultimate goal, e.g. by linking the PRs to a GitHub issue describing the goal?

Reply:
These were just a series of ad-hoc PRs that I created in the context of getting an overview for / understanding the algorithms in `CopyrightStatementsProcessor`, and addressing miscellaneous things I came across while doing so. Eventually, the only "goal" I had was to make the `determineParts()` logic public, so this PR concludes that work. I do not plan to make further `CopyrightStatementsProcessor`-related PRs for the time being.